.github/workflows/fake-bets-mapping-snapshot-file-change.yml

```diff
@@ -0,0 +1,16 @@
+name: 'fake-bets-mapping-snapshot-file-change'
+
+on:
+  workflow_dispatch:
+
+# Don't cancel in progress since we don't want to have half-baked file change snapshot.
+concurrency: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-fake-bets-mapping-snapshot-file-change'
+
+jobs:
+  snapshot:
+    uses: 'abcxyz/pmap/.github/workflows/snapshot-file-change.yml@main' # ratchet:exclude
+    with:
+      WIF_PROVIDER: 'projects/921163060412/locations/global/workloadIdentityPools/github-pool-3b39/providers/github-provider'
+      WIF_SERVICE_ACCOUNT: 'pmap-3b39-ci-sa@pmap-ci.iam.gserviceaccount.com'
+      destination_prefix: 'pmap-dev/mapping'
+      path: 'test/e2e/testdata/fakebets/mapping'
```

Comment:
Just curious, after this change is merged, how the changed file action will work in order to get the already added mapping files?

Reply:
If there is no change, the changed file action won't copy the already added mapping files to gcs. You need to rely on `file-copy` one. If you want to see whether the file change still works, you'll have to change the mapping files, the easier change could be adding annotations.

Reply:
Got it, thx!